.gitignore

```diff
@@ -20,3 +20,5 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+
+tags
```

Comment:
What's tags?

Reply:
I believe it is a result of ctags, vim kept creating it when I opened some files and it was pissing me off :P

Reply:
I see. Perhaps a short comment like the others have would be useful.